distribution/ddf-common/src/main/resources/etc/users.properties

```diff
@@ -32,5 +32,5 @@
 
 # DDF Changes: changed username/pw to admin/admin, added localhost certificate user
 
-admin=admin,group,admin,manager,viewer,system-admin,systembundles
-localhost=localhost,group,admin,manager,viewer,system-admin,system-history,systembundles
\ No newline at end of file
+admin=admin,group,admin,manager,viewer,system-admin,systembundles,ssh
+localhost=localhost,group,admin,manager,viewer,system-admin,system-history,systembundles,ssh
```

Comment:
Should https://github.com/codice/ddf/blob/25955014b299aa57ea6c69967fdbcc9aaf9db462/platform/security/sts/security-sts-propertyclaimshandler/src/test/resources/users.properties be updated as well? @stusison